### ISSUE TITLE
No receipe for Steve's cart coal engine(age 2 to 3 progress item)

### ISSUE THREAD

**Issue by tomm2000** [Darkosto/SevTech-Ages#1731]
There is just no receipe for it, there was one in previous pack version.

**Comment by tomm2000** [comment]
I am using 3.0.6 btw

**Comment by Targren** [comment]
The crafting tab is blank, but there should be a Celestial Crafting Table recipe for it. I created one to progress last night on 3.0.6

**Comment by tomm2000** [comment]
There is none

**Comment by Targren** [comment]
Weird.

**Comment by tomm2000** [comment]
I really want to advance to age 3...I want my immersive engineering

**Comment by Targren** [comment]
Yup. It's missing all right. Have you tried reinstalling the pack? It looks like your scripts may be messed up.

**Comment by tomm2000** [comment]
I may try tomorrow

**Comment by Dobhran** [comment]
We're also having this issue on my server which is running 3.0.6. Three different people cannot craft the Coal Engine in their respective Celestial Altars, and we are now just stuck at the end of Age 2.

**Comment by tomm2000** [comment]
Reinstalling the pack did fix the problem for me...but those who play on a server can't do that...this still need a fix then!

**Comment by tomm2000** [comment]
Turns out it is a problem with resource packs...as I loaded my texture the problem reoccurred...

**Comment by SummitSummit** [comment]
Blast!  I was hoping your solution would fix me being stuck.  Alas, it did not.  My problem is a bit different.  The recipe for the Coal Engine shows up in JEI, but the Celestial Altar will NOT craft it.  I disabled my Resource pack and reloaded the game but it made no difference.  Perhaps I'll back up my save, uninstall the pack and reinstall to see if that fixes it.  I wish there was a way to get just the scripts/configs rather than downloading the entire pack again as I have already had to reinstall due to corrupted scripts with this pack once before.

**Comment by TheLemein** [comment]
It is the resource pack, you have to completely reload in order for it to return

**Comment by SummitSummit** [comment]
Nope.  Just did a completely fresh reinstall of the modpack, loaded it up w/o resource pack, and still can NOT craft Coal Engine.  I suppose I could cheat it in, to see if that will allow me to progress, but I don't want to.  Guess I'll just wait for 3.0.7.

**Comment by Targren** [comment]
Are you sure you have the elements right in the crafting altar?

**Comment by SummitSummit** [comment]
Yup, I've double and triple checked that I have the correct items in the correct spots.  I would take a screenshot but I don't feel like waiting 10 minutes for the pack to load, starting the game and waiting a couple more minutes for it to crash, so I can load it again, wait more minutes, then take a screenshot.
Don't get me wrong, I love the pack, it is amazing, but it is broken right now and I've already wasted 4 hours trouble shooting and don't feel like wasting more time now.

**Comment by Pe4enie** [comment]
@Targren, @SummitSummit hey guys. Just having the very same issue, so can provide for screenshots.
Recipe:
![image](https://user-images.githubusercontent.com/2092103/38706098-471759e6-3eac-11e8-8c26-cecc9893eb71.png)
My setup:
![image](https://user-images.githubusercontent.com/2092103/38706106-5287ddf0-3eac-11e8-9eff-6b1f0a1beae6.png)

As you can see recipe is just not being recognized

**Comment by SummitSummit** [comment]
Yes, that's the exact problem I'm having thanks for the pics.  Can you craft the Starlight Infuser?
